src/sync/init.ts

```diff
@@ -136,8 +150,12 @@ export async function init(session: ISession, opts: Options) {
 
   session.log.info(await FINISHED(session));
```

Comment:
Only show this if `!opts.yes` too, like the welcome message?

Reply:
it's completely silent then but yeah `-y` is for someone who knows what they are doing